packages/devtools_app/lib/src/framework/notifications_view.dart

```diff
@@ -254,6 +254,8 @@ class _NotificationState extends State<_Notification>
               child: Column(
                 crossAxisAlignment: CrossAxisAlignment.start,
                 children: [
+                  if (widget.message.isDismissible)
+                    _DismissAction(widget: widget),
                   _NotificationMessage(widget: widget, context: context),
```

Comment:
can these two items be in a Row so that the text and the dismiss button are in alignment?

Reply:
Done! In working on that, noticed some other weird alignment issues (see PR description for images)